scripts/templates/field_details.j2

```diff
@@ -145,7 +145,9 @@ Note also that the `{{ fieldset['name'] }}` fields are not expected to be used d
 [discrete]
 ==== {{ fieldset['title'] }} Field Usage
 
-For usage and examples of the {{ fieldset['name'] }} fields please see the <<ecs-{{ fieldset['name']|replace("_", "-") }}-usage,usage>> section.
+For usage and examples of the {{ fieldset['name'] }} fields please see the 
```

Comment:
```suggestion
For usage and examples of the {{ fieldset['name'] }} fields, please see the 
```

Not related to any of the changes here, but I think we could add a comma after the intro clause in this sentence?